bin/windows/android-petstore-httpclient.bat

```diff
@@ -1,10 +1,7 @@
 set executable=.\modules\openapi-generator-cli\target\openapi-generator-cli.jar
 
-If Not Exist %executable% (
-  mvn clean package
-)
```

Comment:
Why remove these? Are these lines not working as expected (to build the JAR if not present)?

Reply:
These lines don't work on my Windows machine (I need to run mvnw and it is not on my path anyway) so I removed them, reinstated for people who have a more complete setup.

Reply:
Very good point. May be we should use `mvnw` instead of `mvn` to make it easier for devleopers using Windows machines to build the project.